libbeat/processors/kubernetes/podwatcher.go

```diff
@@ -124,44 +125,64 @@ func (p *PodWatcher) Run() bool {
 	}
 }
 
-func (p *PodWatcher) onPodAdd(pod *corev1.Pod) {
+func (p *PodWatcher) onPodAdd(pod *Pod) {
 	metadata := p.indexers.GetMetadata(pod)
 	p.annotationCache.Lock()
 	defer p.annotationCache.Unlock()
 
-	p.annotationCache.pods[pod.Metadata.GetUid()] = pod
+	p.annotationCache.pods[pod.Metadata.UID] = pod
 
 	for _, m := range metadata {
 		p.annotationCache.annotations[m.Index] = m.Data
 	}
 }
 
-func (p *PodWatcher) onPodUpdate(pod *corev1.Pod) {
-	oldPod := p.GetPod(pod.Metadata.GetUid())
-	if oldPod.Metadata.GetResourceVersion() != pod.Metadata.GetResourceVersion() {
+func (p *PodWatcher) onPodUpdate(pod *Pod) {
+	oldPod := p.GetPod(pod.Metadata.UID)
+	if oldPod.Metadata.ResourceVersion != pod.Metadata.ResourceVersion {
 		//Process the new pod changes
 		p.onPodDelete(oldPod)
 		p.onPodAdd(pod)
 	}
 }
 
-func (p *PodWatcher) onPodDelete(pod *corev1.Pod) {
+func (p *PodWatcher) onPodDelete(pod *Pod) {
 	p.annotationCache.Lock()
 	defer p.annotationCache.Unlock()
 
-	delete(p.annotationCache.pods, pod.Metadata.GetUid())
+	delete(p.annotationCache.pods, pod.Metadata.UID)
 
 	for _, index := range p.indexers.GetIndexes(pod) {
 		delete(p.annotationCache.annotations, index)
 	}
 }
 
+func (p *PodWatcher) getPodMeta(pod *corev1.Pod) *Pod {
+
```

Comment:
Could you please remove the spurious empty lines at the beginning and end of this function?